src/MagnumPlugins/DdsImporter/Test/DdsImporterTest.cpp

```diff
@@ -241,6 +254,153 @@ void DdsImporterTest::dxt5() {
+        {"2DMips_R16G16B16A16_UNORM.dds", PixelFormat::RGBA, PixelType::UnsignedShort},
+        {"2DMips_R16G16_FLOAT.dds", PixelFormat::RG, PixelType::HalfFloat},
+        {"2DMips_R16G16_SINT.dds", PixelFormat::RGInteger, PixelType::Short},
+        {"2DMips_R16G16_SNORM.dds", PixelFormat::RG, PixelType::Short},
+        {"2DMips_R16G16_UINT.dds", PixelFormat::RGInteger, PixelType::UnsignedShort},
+        {"2DMips_R16G16_UNORM.dds", PixelFormat::RG, PixelType::UnsignedShort},
+        {"2DMips_R32_FLOAT.dds", PixelFormat::Red, PixelType::Float},
+        {"2DMips_R32G32B32A32_FLOAT.dds", PixelFormat::RGBA, PixelType::Float},
+        {"2DMips_R32G32B32A32_SINT.dds", PixelFormat::RGBAInteger, PixelType::Int},
+        {"2DMips_R32G32B32A32_UINT.dds", PixelFormat::RGBAInteger, PixelType::UnsignedInt},
+        {"2DMips_R32G32B32_FLOAT.dds", PixelFormat::RGB, PixelType::Float},
+        {"2DMips_R32G32B32_SINT.dds", PixelFormat::RGBInteger, PixelType::Int},
+        {"2DMips_R32G32B32_UINT.dds", PixelFormat::RGBInteger, PixelType::UnsignedInt},
+        {"2DMips_R32G32_FLOAT.dds", PixelFormat::RG, PixelType::Float},
+        {"2DMips_R32G32_SINT.dds", PixelFormat::RGInteger, PixelType::Int},
+        {"2DMips_R32G32_UINT.dds", PixelFormat::RGInteger, PixelType::UnsignedInt},
+        {"2DMips_R32_SINT.dds", PixelFormat::RedInteger, PixelType::Int},
+        {"2DMips_R32_UINT.dds", PixelFormat::RedInteger, PixelType::UnsignedInt},
+        {"2DMips_R8G8B8A8_SINT.dds", PixelFormat::RGBAInteger, PixelType::Byte},
+        {"2DMips_R8G8B8A8_SNORM.dds", PixelFormat::RGBA, PixelType::Byte},
+        {"2DMips_R8G8B8A8_UINT.dds", PixelFormat::RGBAInteger, PixelType::UnsignedByte},
+        {"2DMips_R8G8B8A8_UNORM.dds", PixelFormat::RGBA, PixelType::UnsignedByte},
+        {"2DMips_R8G8B8A8_UNORM_SRGB.dds", PixelFormat::RGBA, PixelType::UnsignedByte},
+        {"2DMips_R8G8_UNORM.dds", PixelFormat::RG, PixelType::UnsignedByte},
+        {"3D_R16G16B16A16_FLOAT.dds", PixelFormat::RGBA, PixelType::HalfFloat},
+        {"3D_R16G16B16A16_SINT.dds", PixelFormat::RGBAInteger, PixelType::Short},
+        {"3D_R16G16B16A16_SNORM.dds", PixelFormat::RGBA, PixelType::Short},
+        {"3D_R16G16B16A16_UINT.dds", PixelFormat::RGBAInteger, PixelType::UnsignedShort},
+        {"3D_R16G16B16A16_UNORM.dds", PixelFormat::RGBA, PixelType::UnsignedShort},
+        {"3D_R32G32B32A32_FLOAT.dds", PixelFormat::RGBA, PixelType::Float},
+        {"3D_R32G32B32A32_SINT.dds", PixelFormat::RGBAInteger, PixelType::Int},
+        {"3D_R32G32B32A32_UINT.dds", PixelFormat::RGBAInteger, PixelType::UnsignedInt},
+        {"3D_R32G32B32_FLOAT.dds", PixelFormat::RGB, PixelType::Float},
+        {"3D_R32G32B32_SINT.dds", PixelFormat::RGBInteger, PixelType::Int},
+        {"3D_R32G32B32_UINT.dds", PixelFormat::RGBInteger, PixelType::UnsignedInt},
+        {"3D_R32G32_FLOAT.dds", PixelFormat::RG, PixelType::Float},
```

Comment:
Just a heads-up: 

- you either need to list all those files explicitly in CMakeLists.txt so they are properly picked up by Android and Emscripten (there is already a list, so just extend it), but I fear this might be too long to fit on the compiler command-line for Emscripten
- or compile them in as a resource